### ISSUE TITLE
`java.lang.reflect.InvocationTargetException`: How to figure out which class is causing this error?

### ISSUE THREAD

**Issue by Agentum07** [nus-cs2103-AY2122S1/forum#233]
Using the latest version of intelliJ on windows 10.
I keep getting a `java.lang.reflect.InvocationTargetException` which is being caused by a `java.lang.NullPointerException` however I cannot figure out which class is causing the error.

![Capture](https://user-images.githubusercontent.com/62797335/133433703-4e7e660f-6a9a-401d-a9a3-befa453b4640.PNG)

I tried running with `--stacktace` and `--scan` but these keep running and never stop.
Is there some other command I can run?

**Comment by damithc** [comment]
@Agentum07 see if clicking on these nodes reveal any additional information.
![image](https://user-images.githubusercontent.com/1673303/133443119-64394704-887f-407d-8ccc-d94c540c7a26.png)

**Comment by aryagiat** [comment]
Hi @Agentum07, try looking at [this](https://stackoverflow.com/questions/6020719/what-could-cause-java-lang-reflect-invocationtargetexception). It might help you debug your issue.

**Comment by marcuspeh** [comment]
Maybe you can printing stack trace of the error and finding the class that is causing the error.

**Comment by Agentum07** [comment]
Thank you for all your help fellow peers and profs! I figured out my mistake. I was calling an arraylist before initializing it.
I understood my mistake by eyeballing my code, so my question still stands. However, I will close this issue.

Prof @damithc clicking at those nodes did not reveal more information. They just collapsed the menu, basically acted as arrows.
@marcuspeh as mentioned in the question, I already ran `stack trace` it was not of any help.
@aryagiat thank you for the link!
